add python host/run dependency

bollwyvl opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Changes:
- [x] update host/run deps
  - [x] `python`
- [ ] add arches
  - [x] `osx-64`
  - [x] `win-64`
  - [ ] `osx-arm64`
  - [ ] `linux-aarch64`
- [x] install python packages
  - [x] `swi_janus`
  - [x] `swiplserver`
- [x] add licenses for vendored code
- [ ] cleanup
  - [ ] revert `skip` of non-min python

bollwyvl committed:
add python to host/build matrix

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
add python 3.14

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
try some cmake

bollwyvl committed:
be more explicit about build script

bollwyvl committed:
more linkage errors

bollwyvl committed:
remove unused cmake args

bollwyvl committed:
try minimum support cmake

bollwyvl committed:
dig into python

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
roll back abi3 for now

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
restore linux

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
more abi3 removal

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
update license

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
try some arches

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.12.18.57.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
more platform work

bollwyvl committed:
more license

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19334131147](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19334131147). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
move an x11 host dep from linux to unix

bollwyvl committed:
more cmake invocation

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ Whenever possible fix all shellcheck findings ('shellcheck --enable=all --shell=bash --exclude=SC2154 recipe/*.sh -f diff | git apply' helps)
 * ℹ️ 
 * ℹ️ In /tmp/tmpw07_8c9r/cf_feedstock_ops_dir/recipe/build-swi-prolog.sh line 47:
 * ℹ️         --target install
 * ℹ️         ^------^ SC2215 (warning): This flag is used as a command name. Bad line break or missing [ .. ]?
 * ℹ️ 
 * ℹ️ For more information:
 * ℹ️   https://www.shellcheck.net/wiki/SC2215 -- This flag is used as a command na...


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19335102354](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19335102354). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
more x11 for osx

bollwyvl committed:
try without win stdlib

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl committed:
try explicit cmake cxx

bollwyvl committed:
try newer c_stdlib on osx

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.13.12.38.23

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
try more build stuff

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.13.12.38.23

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
put back a flag

bollwyvl committed:
try gmp on win

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.13.20.21.10

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
rely on libwinpthread from gmp

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19346624700](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19346624700). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
also remove pthreads-win32 from run

bollwyvl committed:
try more gmp on win

bollwyvl committed:
gmp win as env vars

bollwyvl committed:
add winpthreads to host

bollwyvl committed:
try winpthreads-devel

bollwyvl committed:
roll back some win

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.14.11.30.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
clean up run deps

bollwyvl committed:
trim down builds

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.14.11.30.28

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
remove unused -j option for win

bollwyvl committed:
try win CMAKE_CXX_STANDARD=14

bollwyvl committed:
try win CMAKE_CXX_STANDARD=11

bollwyvl committed:
try m2

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.17.12.05.48

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
try m2w64

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.17.12.05.48

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
try wrapper script

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.17.12.05.48

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl commented:
@conda-forge-admin please rerender

bollwyvl committed:
back to just gmp

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19434354769](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19434354769). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
use full posix on win

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.17.12.05.48

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ Use of `posix` package on windows is deprecated. Use `m2-base`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19434482019](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19434482019). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
only skip ossuuid on osx_arm64

bollwyvl committed:
use m2-base

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl committed:
copy win sh to src_dir

bollwyvl committed:
no executable bit

bollwyvl committed:
fix script name

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ Whenever possible fix all shellcheck findings ('shellcheck --enable=all --shell=bash --exclude=SC2154 recipe/*.sh -f diff | git apply' helps)
 * ℹ️ 
 * ℹ️ In /tmp/tmpxd8cp35r/cf_feedstock_ops_dir/recipe/build-swi-prolog.sh line 22:
 * ℹ️     if [[ "$CONDA_BUILD_CROSS_COMPILATION" == 1 ]]; then
 * ℹ️            ^----------------------------^ SC2250 (style): Prefer putting braces around variable references even when not strictly required.
 * ℹ️ 
 * ℹ️ Did you mean: 
 * ℹ️     if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == 1 ]]; then
 * ℹ️ 
 * ℹ️ For more information:
 * ℹ️   https://www.shellcheck.net/wiki/SC2250 -- Prefer putting braces around vari...


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19435801388](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19435801388). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
linting

bollwyvl committed:
add default value to check for cross-compiling on osx

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl committed:
more win env vars

bollwyvl committed:
try cygpath

bollwyvl committed:
try win PATH rewriting

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ Whenever possible fix all shellcheck findings ('shellcheck --enable=all --shell=bash --exclude=SC2154 recipe/*.sh -f diff | git apply' helps)
 * ℹ️ 
 * ℹ️ In /tmp/tmpzl3chwwl/cf_feedstock_ops_dir/recipe/build-swi-prolog-win.sh line 4:
 * ℹ️ [ -n "$PATH_OVERRIDE" ] && export PATH="$PATH_OVERRIDE"
 * ℹ️ ^---------------------^ SC2292 (style): Prefer [[ ]] over [ ] for tests in Bash/Ksh.
 * ℹ️       ^------------^ SC2250 (style): Prefer putting braces around variable references even when not strictly required.
 * ℹ️                                         ^------------^ SC2250 (style): Prefer putting braces around variable references even when not strictly required.
 * ℹ️ 
 * ℹ️ Did you mean: 
 * ℹ️ [[ -n "${PATH_OVERRIDE}" ]] && export PATH="${PATH_OVERRIDE}"
 * ℹ️ 
 * ℹ️ For more information:
 * ℹ️   https://www.shellcheck.net/wiki/SC2250 -- Prefer putting braces around vari...
 * ℹ️   https://www.shellcheck.net/wiki/SC2292 -- Prefer [[ ]] over [ ] for tests i...


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19436743532](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19436743532). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
try less on win PATH

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl committed:
try patching out cmake PATH check on win

bollwyvl committed:
fix cross-compiling cmake args

bollwyvl committed:
try c_stdlib for osx-arm64

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.1 and conda-forge-pinning 2025.11.17.12.05.48

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
roll back osx, don't pgo in emulation

bollwyvl committed:
merge upstream

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.02.23.00.48

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
fix version
